mage_ai/tests/data_cleaner/statistics/test_calculator.py

```diff
@@ -70,11 +70,7 @@ def test_calculate_statistics_overview(self):
         self.assertEqual(data['cancelled/quality'], 'Good')
 
     def test_calculate_statistics_overview_divide_by_zero(self):
-        calculator = StatisticsCalculator(
-            column_types=dict(
-                age='number',
-            ),
-        )
+        calculator = StatisticsCalculator(column_types=dict(age='number',),)
```

Comment:
if you change it to one line, no need to add comma after 'number' and ')'

Reply:
Resolved